<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskPocketParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskPocketParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="Gui::PrefQuantitySpinBox" name="lengthEdit2" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelOffset">
       <property name="text">
        <string>Offset to face</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonFace">
       <property name="text">
        <string>Face</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Gui::PrefQuantitySpinBox" name="lengthEdit" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
       <property name="minimum" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="changeMode">
       <item>
        <property name="text">
         <string>Dimension</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelLength">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Gui::PrefQuantitySpinBox" name="offsetEdit" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineFaceName"/>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkFaceLimits">
       <property name="text">
        <string>Check face limit</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelLength2">
       <property name="text">
        <string>2nd length</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBoxMidplane">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Symmetric to plane</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBoxReversed">
       <property name="text">
        <string>Reversed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUsePipe">
     <property name="toolTip">
      <string>Enable this option to use pipe (i.e. sweep) operation to create draft angles.
If disabled, then use loft operation instead. Pipe operation can avoid some
unnecessary use of BSplineSurface for planar faces in the resulting shape.</string>
     </property>
     <property name="text">
      <string>Use pipe operation for draft</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Taper</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="labelTaperAngle">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::PrefQuantitySpinBox" name="taperAngleEdit" native="true">
          <property name="keyboardTracking" stdset="0">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>deg</string>
          </property>
          <property name="maximum" stdset="0">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep" stdset="0">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="labelTaperAngle2">
          <property name="text">
           <string>2nd Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::PrefQuantitySpinBox" name="taperAngleEdit2" native="true">
          <property name="keyboardTracking" stdset="0">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>deg</string>
          </property>
          <property name="maximum" stdset="0">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep" stdset="0">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="autoInnerTaperAngle">
        <property name="text">
         <string>Auto inner angle</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="labelInnerTaperAngle">
          <property name="text">
           <string>Inner angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::PrefQuantitySpinBox" name="innerTaperAngleEdit" native="true">
          <property name="keyboardTracking" stdset="0">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>deg</string>
          </property>
          <property name="maximum" stdset="0">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep" stdset="0">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="labelInnerTaperAngle2">
          <property name="text">
           <string>2nd inner angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Gui::PrefQuantitySpinBox" name="innerTaperAngleEdit2" native="true">
          <property name="keyboardTracking" stdset="0">
           <bool>false</bool>
          </property>
          <property name="unit" stdset="0">
           <string>deg</string>
          </property>
          <property name="maximum" stdset="0">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep" stdset="0">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefQuantitySpinBox</class>
   <extends>Gui::QuantitySpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>changeMode</tabstop>
  <tabstop>lengthEdit</tabstop>
  <tabstop>offsetEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
